src/features/user/ManageProjects/index.tsx

```diff
@@ -18,7 +18,13 @@ import { useTranslation } from 'next-i18next';
 import { ErrorHandlingContext } from '../../common/Layout/ErrorHandlingContext';
 import { handleError, APIError } from '@planet-sdk/common';
 
-export default function ManageProjects({ GUID, token, project }: any) {
+interface Props {
+  GUID: string;
+  token: string | null;
+  project: any;
+}
```

Comment:
1. Judging from the way this component is used, `GUID` does not seem to be always needed, nor does `project`.
2. Types for `project` are defined in `@planet-sdk`. I think one of the predefined types would work here (either the `ProjectDefault` type or `ProjectExtended`).

Reply:
I've made a small change to remove `{}` as a possible value for `project`. The TS warnings at the places `ManageProjects` is being used can be addressed later.